### DISCUSSION TITLE
Drupal plugin error:Cannot load the "mosparo_connection" entity with NULL ID.

### DISCUSSION THREAD

**Discussion by 0MAL** [mosparo/mosparo#200] [Problems & Issues]
<img width="1300" alt="screenshot1203" src="https://github.com/mosparo/mosparo/assets/91258926/b98c663f-db95-4d6f-9790-5bc84d045549">

I installed mosparo module and activated, then set mosparo connections at admin/config/mosparo/connections.
Then I added a new contact form and add field of mosparo, then error occurs.
Drupal 10.1.8

**Comment by zepich** [comment]
Hi @0MAL 

Thank you very much for your report.

I tested it with my Drupal installation, but I don't have this issue and cannot explain this error right now. 

I will set up a new Drupal and test it there again.

I will inform you as soon as I've tested it.

Kind regards,

zepich

**Comment by zepich** [comment]
Hi @0MAL 

I've installed a new Drupal (version 10.1.8). After that, I added the mosparo Integration module and enabled the Base Integration and the Integration for Contact submodule.

After that, I went to the mosparo Connections in the Drupal settings and added a new connection like this (of course, with Public and Private keys)

![image](https://github.com/mosparo/mosparo/assets/391914/57f091f6-cbcc-48f3-8da6-5acb289349a0)

After that, I went to Structure -> Contact forms and created a new form. In this new form, I added the mosparo field. I entered a name and selected the connection (which is preselected).

![image](https://github.com/mosparo/mosparo/assets/391914/0774a282-62eb-43ce-af00-755e082d6ea9)

After saving the form, I open the view of the form, and then the mosparo box is visible at the bottom.

![image](https://github.com/mosparo/mosparo/assets/391914/39d9eb04-b6a3-4c8e-b592-3b62748970f1)

Are those the same things you did? As far as I understand, you received the error after adding the field to the form. Is that correct?

You're using the mosparo Integration module v1.0, correct?

What PHP version do you use?

Kind regards,

zepich

**Comment by 0MAL** [comment]
I'm using mosparo Integration module v1.0, php version is 8.1.27.
The mosparo box is not visible for me, when I open the view of the form, error is displayed on the screen instead.
Anyway, Drupal's module page says "This project is not covered by Drupal’s [security advisory policy](https://www.drupal.org/security-advisory-policy)." and I have alternative ways, so  I won't use this module at least until this passing Drupals security policy.

**Comment by zepich** [comment]
Okay, thank you very much for your update.

Thank you very much for reporting this issue. I will have a look at it anyways.
